### DISCUSSION TITLE
Github Projects - Detailed Notes

### DISCUSSION THREAD

**Discussion by ihtasham42** [community/community#4603] [Projects and Issues]
It would be nice if user could expand each note within Github Projects, opening up a 'modal' where they can add more description. This very feature is evident in Trello after clicking on a 'card'.

**Comment by Lucaffo** [comment]
I would suggest to convert the card in an issue like that: 
![image](https://user-images.githubusercontent.com/55745404/125215058-7c2c3b00-e2ba-11eb-9e4d-97bd9579baa1.png)
in this way, you have a good issue management linked to your projects and more options to write descriptions and things.

![image](https://user-images.githubusercontent.com/55745404/125215143-ce6d5c00-e2ba-11eb-9b79-f72c9a80140d.png)
As you can see "No description provided" is your card content. 

Anyway, you can even use Markdown inside the card content but it's quite odd and ugly.

![image](https://user-images.githubusercontent.com/55745404/125215255-2dcb6c00-e2bb-11eb-98ad-52792b9d31ba.png)

**Reply by ihtasham42** [reply]
Thank you for the solution.
